### PR TITLE
enable various tests that need cunit

### PR DESCRIPTION
**Pull request by mikoxyz**
depends on #1394

### CODE REVIEW

**Comment by nekopsykose** [comment]
i already packaged cunit in https://github.com/chimera-linux/cports/pull/1394

File: main/libtracefs/patches/0002-libtracefs-utest-Define-_LARGEFILE64_SOURCE-for-lsee.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+diff --git a/utest/tracefs-utest.c b/utest/tracefs-utest.c
+index 07ecd32..737e54a 100644
+--- a/utest/tracefs-utest.c
++++ b/utest/tracefs-utest.c
+@@ -3,6 +3,8 @@
+  * Copyright (C) 2020, VMware, Tzvetomir Stoyanov <tz.stoyanov@gmail.com>
+  *
+  */
++#define _LARGEFILE64_SOURCE
++
```
Comment (nekopsykose) [inline main/libtracefs/patches/0002-libtracefs-utest-Define-_LARGEFILE64_SOURCE-for-lsee.patch:28]: this is the wrong fix, it should remove lseek64 and use lseek + -D_FILE_OFFSET_BITS=64(for glibc; on musl it's always 64) instead

**Comment by mikoxyz** [comment]
On Fri Feb 23, 2024 at 2:11 AM CET, alice wrote:
> this is the wrong fix, it should remove lseek64 and use lseek + -D_FILE_OFFSET_BITS=64(for glibc; on musl it's always 64) instead
meh, this patch was what got merged upstream, so ¯\_(ツ)_/¯

**Comment by nekopsykose** [comment]
write the good one next time then :P

**Pull request merged**